test/e2e/run_privileged_test.go

```diff
@@ -69,7 +69,7 @@ var _ = Describe("Podman privileged container tests", func() {
 	})
 
 	It("podman non-privileged should have very few devices", func() {
-		session := podmanTest.Podman([]string{"run", "busybox", "ls", "-l", "/dev"})
+		session := podmanTest.Podman([]string{"run", "-t", "busybox", "ls", "-l", "/dev"})
```

Comment:
What broke about this that it needs `-t` now?

Reply:
evidently the capturing of the output

Reply:
Huh. We should test this more at some point and figure out why. But later.

Reply:
With this patch what does
```
podman run busybox ls -1 /dev
```
output?